cdap-docs/developers-manual/source/building-blocks/datasets/fileset.rst

```diff
@@ -356,7 +356,7 @@ will be registered with the output ``PartitionedFileSet`` at the end of the job.
 Note that any partitions written to must not previously exist. Otherwise, the MapReduce job will fail at the
 end of the job and none of the partitions will be added to the ``PartitionedFileSet``.
 
-Incrementally Processing PartitionedFileSets
+Incrementally Processing PartitionedFileSets (MapReduce)
```

Comment:
@awholegunch is going to break it up into a broader section for Incremental processing and then have these two as sub-sections of that.